lib/watcher/cpu-profile.js

```diff
@@ -5,7 +5,7 @@
 
 'use strict';
 
-var agent = require('../agent');
+var agent = require('../agent')();
```

Comment:
need to do lazy evaluation of  this, we'll have to figure out a way to make sure appmetrics is not required if its not intended to be used